### PR TITLE
(fix) Tweak background colour of the forms list searchbox 

### PR DESCRIPTION
**Pull request by jwnasambu**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Changed the background color of the search-bar from white to `$ui-01` basing on this document https://www.loom.com/share/a62a82abb3324c8bb62835098a34e0b6

## Screenshots

**This is the UI before the changes made**
Screenshot 2023-08-15 at 4 12 49 PM

**This is the UI after making changes**

Screenshot 2023-08-15 at 4 15 07 PM

### CODE REVIEW

File: src/components/dashboard/dashboard.scss [hunk 1]
```diff
@@ -65,9 +65,9 @@
   :global(.cds--table-toolbar) {
     position: relative;
     height: 2rem;
-    min-height: 0rem;
     overflow: visible;
     top: 0;
+    background-color: $ui-01;
```
Comment (denniskigen) [inline src/components/dashboard/dashboard.scss:70]: How about a different approach that won't require removing the `min-height`? I suggest adding a class named `headerContainer` to the `TableToolbarContent` component with the following rules:

```scss
.headerContainer {
  background-color: colors.$gray-10;
}	
```

This targets the container only without needing to affect other elements.

**Comment by jwnasambu** [comment]
@denniskigen Kindly am pushing the changes shortly!

**Comment by jwnasambu** [comment]
@denniskigen thanks for the review! I have pushed the proposed changes!

**Comment by hadijahkyampeire** [comment]
@denniskigen from what @jwnasambu mentioned, this work has already been pushed in another PR, so I will go ahead and close this one. Thanks @jwnasambu for the work.

**Pull request closed without merging**